staging-service/pom.xml

```diff
@@ -83,6 +83,10 @@
             <groupId>org.springframework</groupId>
             <artifactId>spring-beans</artifactId>
         </dependency>
+        <dependency>
+            <groupId>org.springframework</groupId>
+            <artifactId>spring-core</artifactId>
```

Comment:
No need for this dependency if you use pass StagingProperties to the constructor of StartupListener

Reply:
Removed